internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -27,12 +29,86 @@ func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchive_basic(t
 		CheckDestroy:             checkDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: resourceConfigBasic(projectID, endpointID),
+				Config: resourceConfigBasic(projectID, endpointID, defaultComment),
 				Check: resource.ComposeTestCheckFunc(
 					checkExists(resourceName),
 					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
 					resource.TestCheckResourceAttr(resourceName, "endpoint_id", endpointID),
-					resource.TestCheckResourceAttrSet(resourceName, "comment"),
+					resource.TestCheckResourceAttr(resourceName, "comment", defaultComment),
+					resource.TestCheckResourceAttrSet(resourceName, "type"),
+					resource.TestCheckResourceAttrSet(resourceName, "provider_name"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportStateIdFunc: importStateIDFunc(resourceName),
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchive_updateComment(t *testing.T) {
+	// Skip because private endpoints are deleted daily from dev environment
+	acc.SkipTestForCI(t)
```

Comment:
similar comment, were you able to run all skipped tests in local?

Reply:
@lantoli curious to know your opinion: is there an opportunity to actually convert this to run in CI? not sure how big the work is

Reply:
wondering if this is something that Espen could do it (not advocating for it)

Reply:
we used to have them enabled in cloud-dev. We disabled them in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1929 because cloud-dev started to delete the private endpoint every day similar to the cluster.

Reply:
for example we could reuse a project that has cleanup skipped

Reply:
All that would be required AFAIK is having access to a `MONGODB_ATLAS_PROJECT_ID`, `MONGODB_ATLAS_ORG_ID`, `MONGODB_ATLAS_PRIVATE_ENDPOINT_ID`. The VPC Endpoint itself is not deleted in the AWS account. 
We'll address that in the test project :)